Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -50,13 +50,12 @@ GameFile::GameFile(const QString& path) : m_path(path)
 
   if (!TryLoadCache())
```

Comment:
and i think you would have to make it `if (TryLoadCache() || TryLoadVolume() || TryLoadElfDol()) { m_valid = true; }` to make it equivalent.